libs/ui/src/lib/search-hit/search-hit.module.css

```diff
@@ -45,3 +45,23 @@
   color: #969ba0;
   font-size: 16px;
 }
+
+@media (max-width: 500px) {
```

Comment:
320px — 480px: Mobile devices
481px — 768px: iPads, Tablets
769px — 1024px: Small screens, laptops
1025px — 1200px: Desktops, large screens
1201px and more —  Extra large screens, TV